webservices/legal_docs/current_cases.py

```diff
@@ -661,23 +620,55 @@ def get_adr_case_status(case_id):
 def get_mur_dispositions(case_id):
     with db.engine.connect() as conn:
         rs = conn.execute(MUR_ADR_DISPOSITION_DATA.format(case_id))
+
+        # Get the allowed displayed MUR disposition category list from table fecmur.ref_case_disposition_category
+        category_list = [dict(row) for row in conn.execute(CASE_DISPOSITION_CATEGORY, "MUR")]
+        # print(category_list):
```

Comment:
@fec-jli can you remove all the commented lines from this file?

Reply:
Good catch, Thanks @pkfec